src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types';
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+type Props = {
+  todo: Todo;
+};
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <td
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      {todo.user && <UserInfo user={todo.user} />}
```

Comment:
Ensure that `todo.user` is always defined before rendering `UserInfo`. Consider adding a fallback or a check to handle cases where `todo.user` might be undefined to prevent runtime errors.